packages/oracle-node/src/config.ts

```diff
@@ -66,6 +66,8 @@ const DEFAULT_OPTIMISM_RPC_URLS = [
 ];
 const DEFAULT_HISTORICAL_DATA_PACKAGES_URL =
   "https://oracle-gateway-1.b.redstone.finance";
+const DEFAULT_BINANCE_WEBSOCKET_API_URL =
+  "wss://testnet.binance.vision/ws-api/v3";
```

Comment:
For some reason, the mainnet URL (`wss://ws-api.binance.com:443/ws-api/v3`) is not working for me, so I used the testnet as an example. This is a recruitment task, after all, so I think the code is more important than the data source. However, I can investigate further to determine why it doesn't work if you want.